### PR TITLE
release: 0.0.3

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,61 @@
...
+
+### Fixes
+
+- **Improved configuration support for non-cargo dist users - [pomdtr]/[#262] , [ashleygwilliams]/[↬281]**
+
+    Previously, setting `cargo-dist` as false, or omitting it should have been
+    sufficient to stop oranda from attempting to parse your releases as
+    cargo-dist artifacts, however `cargo-dist: false` did not work! How the
+    entire artifacts config object is handled has been reworked and tested.
+    Addiitionally, work from a refactor has also made project with a mixture of
```
Comment (manyinsects) [inline CHANGELOG.md]:
```suggestion
    Additionally, work from a refactor has also made projects with a mixture of
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,61 @@
...
+
+    When you use cargo-dist, we display a header on your index page that
+    detects your user's platform and recommends and installer to them. We did
+    not previously have a fallback if we detected a system that none of the
+    installers supported. Now, in that scenario- we'll offer an artifact
+    download as the header option.
+
+- **Dev commands has proper default values: [ashleygwilliams]/[#256],[↬260]**
+
+    Due to a false hope that `#[derive(Default)]` would collect default from
```
Comment (manyinsects) [inline CHANGELOG.md]:
```suggestion
    Due to a false hope that `#[derive(Default)]` would collect defaults from 
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,61 @@
...
+    When you use cargo-dist, we display a header on your index page that
+    detects your user's platform and recommends and installer to them. We did
+    not previously have a fallback if we detected a system that none of the
+    installers supported. Now, in that scenario- we'll offer an artifact
+    download as the header option.
+
+- **Dev commands has proper default values: [ashleygwilliams]/[#256],[↬260]**
+
+    Due to a false hope that `#[derive(Default)]` would collect default from 
+    the `clap` derive API, we shipped the `dev` command with null defaults.
```
Comment (manyinsects) [inline CHANGELOG.md]: I would reword the end of this, maybe "with no defaults".

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,61 @@
...
+    `dev` now has the same defaults that `serve` does, as is to be expected.
+
+- **Improved typogrphy and layout styles on mobile = [ashleygwilliams]/[#234], [SaraVieira]/[↬276]**
+
+### Maintenance
+
+- **Data fetching refactor: [ashleygwilliams]/[#226],[↬274]**
+- **Update actions/checkout to v3: [ashleygwilliams]/[#251],[↬255]**
+
+[#209]: https://github.com/axodotdev/oranda/issues/209
```
Comment (manyinsects) [inline CHANGELOG.md:56]: (nag) why are some of these with the `#` symbol and some of them with the `↬` symbol?

Comment (ashleygwilliams) [reply CHANGELOG.md:56]: some are issues and some a PRs ;) - i do think because i wanna celebrate when an external contributor files a good issue

File: CHANGELOG.md (hunk 3 above)
Comment (ashleygwilliams) [reply CHANGELOG.md]: ah- so there are defaults ... like it "works" but if you derive default on a struct it functionally sets everything to the "null" option (0, none, etc)

File: CHANGELOG.md (hunk 4 above)
Comment (manyinsects) [reply CHANGELOG.md:56]: ah okay, fair enough, shakes fist at github for not differentiating there

File: CHANGELOG.md (hunk 3 above)
Comment (manyinsects) [reply CHANGELOG.md]: hm, but then i struggle to see how this contributes to the end user experience here... i still think "null defaults" sounds kinda weird. is "nulled" a real word we can use here?

Comment (manyinsects) [reply CHANGELOG.md]: or actually maybe just put `null` in backticks so people know it's a computer sciencey term instead of the uh, number i guess

Comment (manyinsects) [reply CHANGELOG.md]: OH i guess that's why it's so weird to me - null means zero in german lol

Comment (ashleygwilliams) [reply CHANGELOG.md]: i can update this to be more specific!!

File: CHANGELOG.md (hunk 4 above)
Comment (ashleygwilliams) [reply CHANGELOG.md:56]: yeah usually i will write out issue or pr or something else but a previous colleague used the little arrow and i kinda liked it- it is true it's not exactly obvious tho
